Octokit/Models/Request/Enterprise/StartMigrationRequest.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Request body for starting a migration.
+    /// </summary>
+    /// <remarks>
+    /// See <a href="https://developer.github.com/v3/migration/migrations/#start-a-migration">docs</a>
+    /// for more information.
+    /// </remarks>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class StartMigrationRequest
+    {
+        /// <summary>
+        /// Parameter-less constructor needed for SimpleJsonSerializer.
+        /// </summary>
+        public StartMigrationRequest()
+        { }
+
+        public StartMigrationRequest(
+            IReadOnlyList<string> repositories
+            ) :
+            this(repositories, false, false)
+        { }
+
+        public StartMigrationRequest(
```

Comment:
That was added because the `CanBeDeserialized` test was failing. You told me about this fix, remember?